system/datatypes/peer_reviewed_article.php

```diff
@@ -618,10 +634,8 @@ private function updateDiscussionsContainer($type, $discussionsContainer) {
 			}
 		}
 
-		if (self::STATUS_PUBLISHED !== $status) {
-			$commentForm = $this->createDiscussionForm($type);
-			$discussionsContainer->append($commentForm->elem->children());
-		}
+		$commentForm = $this->createDiscussionForm($type);
```

Comment:
I agree with this change, it was implemented like this on request.. Perhaps a discussion is needed to see what is the proper implementation.